src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,41 @@
+import java.util.Arrays;
+
+public class StringAddCalculator {
+    public static Number splitAndSum(String sentence) {
+        if (!validateSentence(sentence))
+            return new Number(0);
+
+        return calculate(sentence);
+    }
+
+    private static Number calculate(String sentence) {
+        CustomSentenceDivider customSentenceDivider = new CustomSentenceDivider(sentence);
+        if (customSentenceDivider.isCustomExpression()) {
+            Expression expression = getExpFromCustomSplitterDivider(customSentenceDivider);
+            return expression.sum();
+        }
+
+        Expression expression = new Expression(sentence, new Splitters());
+        return expression.sum();
+    }
+
+    private static Expression getExpFromCustomSplitterDivider(CustomSentenceDivider customSentenceDivider) {
+        Splitters splitters = new Splitters(Arrays.asList(customSentenceDivider.getSplitter()));
+        return new Expression(customSentenceDivider.getExpressionString(), splitters);
+    }
+
+    private static boolean validateSentence(String sentence) {
+        if (checkNull(sentence))
```

Comment:
```suggestion
        if (Objects.isNull(sentence))
```
Objects# 에 대해 알아보고 활용해보세요 😄